app/pages/sections/guarantor/GuarantorRequiredPage.scala

```diff
@@ -14,19 +14,14 @@
  * limitations under the License.
  */
 
-package pages.sections.consignee
+package pages.sections.guarantor
 
-import pages.behaviours.PageBehaviours
+import pages.QuestionPage
+import play.api.libs.json.JsPath
 
-class ConsigneeExcisePageSpec extends PageBehaviours {
+case object GuarantorRequiredPage extends QuestionPage[Boolean] {
 
-  "ConsigneeExcisePage" - {
+  override def path: JsPath = JsPath \ toString
```

Comment:
`override def path: JsPath = JsPath \ "guarantor" \ toString`

Reply:
Good spot. Probably not for this PR, but should this change be added to the `move2section.sh` script?

Reply:
We could do, although that `move2section.sh` is quite basic at the moment of moving stuff to packages and not doing and processing of the files.